cirq-ft/cirq_ft/algos/arithmetic_gates.py

```diff
@@ -130,6 +131,147 @@ def _t_complexity_(self) -> infra.TComplexity:
         )
 
 
+@attr.frozen
+class BiQubitsMixer(infra.GateWithRegisters):
+    """Implements the COMPARE2 (Fig. 1) https://www.nature.com/articles/s41534-018-0071-5#Sec8
```

Comment:
this links to the acknowledgements section lol. Please remove the `#Sec8`.

Reply:
`sec8` is the "Electronic supplementary material" which is where the decomposition is. it's not part of the main paper but is in the supplementary material